package.json

```diff
@@ -21,7 +21,8 @@
     "framer-motion": "^6.5.1",
     "next": "^12.3.1",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
```

Comment:
I added react-firebase-hooks to the npm packages,
so we will have to reinstall npm